clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListCatalogProperties.java

```diff
@@ -49,31 +48,19 @@ public ListCatalogProperties(CommandContext context, String metalake, String cat
   /** List the properties of a catalog. */
   @Override
   public void handle() {
-    Catalog gCatalog = null;
+    try {
+      GravitinoClient client = buildClient(metalake);
+      Catalog gCatalog = client.loadCatalog(catalog);
 
-    try (GravitinoClient client = buildClient(metalake)) { // Ensures client is closed
-      gCatalog = client.loadCatalog(catalog);
+      if (gCatalog != null) {
+          printProperties(gCatalog.properties());
+      }
```

Comment:
Please move this out of the try block, adding it here makes it unclear what call throws what exception.

Reply:
This has not been reloved.